lib/broccoli/angular2-app.js

```diff
@@ -87,10 +87,16 @@ Angular2App.prototype.toTree = function () {
     destDir: 'vendor'
   });
 
+  var publicDirTree = new Funnel(path.join(sourceDir, 'public'), {
```

Comment:
My bad, I talked to the current owner of this feature so we knew :)

I would like to move the content of the `public` folder inside the `dist`, yes. Not make a `dist/public`, but rather have `public/**/*` copied to `dist/**/*`.